openedx/core/djangoapps/schedules/admin.py

```diff
@@ -9,22 +13,87 @@ class ScheduleExperienceAdminInline(admin.StackedInline):
     model = models.ScheduleExperience
 
 
+def _set_experience(db, human, modeladmin, request, queryset):
```

Comment:
Where does the `modeladmin` parameter get passed into the partial?  I see how `request` and `queryset` must be getting passed in by django admin (akin to the other actions like `activate_schedules`).

Reply:
`modeladmin`, `request`, and `queryset` are all passed to all registered actions. When the actions are methods, the `modeladmin` argument is typically named `self` instead.

Reply:
ah.  forgot about `self`.  Got it.  thanks.